hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneDirectory.java

```diff
@@ -150,6 +150,7 @@ private void setupOzoneFileSystem()
     OzoneConfiguration conf = new OzoneConfiguration();
     conf.setInt(FS_TRASH_INTERVAL_KEY, 1);
     conf.set(OMConfigKeys.OZONE_OM_LAYOUT_VERSION, "V1");
+    conf.setBoolean(OMConfigKeys.OZONE_OM_ENABLE_FILESYSTEM_PATHS, true);
```

Comment:
We should add a function in OzoneTestUtils to enabled fsPaths.
Something like optimizeFSPaths(conf)

Reply:
I have add OzoneTestUtils.configureFSOptimizedPaths, which is visible to all `integration-test` cases. Also, I've add the same function into TestOmRequestUtils.configureFSOptimizedPaths, this is basically added to avoid adding new project dependency to `integration-test`. All the `ozone-manager` project test cases using `TestOmRequestUtils` function. 

Do you agree to this changes ?

Reply:
I have removed duplicate method from OzoneTestUtils  and retained TestOmRequestUtils#configureFSOptimizedPaths